tests/playbooks/tests_bond_options.yml

```diff
@@ -83,6 +83,7 @@
             /sys/class/net/{{ controller_device }}/bonding/'{{ item.key }}'
           name: "** TEST check bond settings"
           register: result
+          changed_when: false
```

Comment:
The `changed_when` changes also seem to be a good candidate for one commit to be merged sooner.